src/components/shared/SearchInput.js

```diff
@@ -47,10 +47,22 @@ class SearchInput extends Component {
 
   componentDidMount() {
     this.$input.focus();
+    if (this.$input.value != "") {
+      this.$input.setSelectionRange(
+        this.$input.value.length + 1,
+        this.$input.value.length + 1
+      );
+    }
   }
 
   componentDidUpdate() {
     this.$input.focus();
+    if (this.$input.value != "") {
+      this.$input.setSelectionRange(
+        this.$input.value.length + 1,
+        this.$input.value.length + 1
```

Comment:
what does this do?

Reply:
Without this when it opens with the modifiers (`@`/`#`etc) the cursor would be before the modifier and not after it.